webui/react/src/e2e/models/common/base/BasePage.ts

```diff
@@ -48,7 +48,6 @@ export abstract class BasePage implements ModelBasics {
     if (verify) {
       // TODO this does nothing because we end up checking, passing, and then getting redirected
       await this._page.waitForURL(this.url);
```

Comment:
part of the `toHaveDeterminedTitle` change. we dont need the commented code anymore.